.github/workflows/validatecrd.yml

```diff
@@ -0,0 +1,18 @@
+name: validate crd
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nolar/setup-k3d-k3s@v1
+        with:
+          version: v1.26.5
```

Comment:
this mean we have to upgrade k3s version manually?

Reply:
that's the k8s version. I was wondering if this is confusing and if a comment could help. or to put it in other words: we are in control what version of k8s is used.

Reply:
I added a comment to make the purpose of this version clear.